osclib/stagingapi.py

```diff
@@ -110,6 +101,20 @@ def __getattr__(self, attr):
         # Raise AttributeError like normal.
         return self.__getattribute__(attr)
 
+    # lazy load
+    def rings(self):
```

Comment:
Uhm no, this is a bug as Python is not a Lisp 2 -a-like language:

```
>>> class A:
...   def __init__(self):
...     self.a = 0
...   def a(self):
...     if self.a:
...        print 'no zero'
... 
>>> kk = A()
>>> kk.a
0
>>> kk.a()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'int' object is not callable
```

Rename it to _rings(), or get_rings()

Reply:
or use \@property ?

Reply:
ok, I'm out

Reply:
https://github.com/openSUSE/openSUSE-release-tools/blob/master/ReviewBot.py#L152